modules/nodes-adding-bare-metal-host-to-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-web-console_{context}"]
+= Adding a bare metal host to the cluster using the web console
+
+You can add bare metal hosts to the cluster in the web console.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
```

Comment:
Same comment as the other section - provisioning a host for use in the cluster is what this section is describing I think?

Reply:
What about: 

`* Install a RHCOS compute machine on bare metal infrastructure for use in the cluster.`